squidb-annotations/src/com/yahoo/squidb/annotations/defaults/DefaultNull.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the Apache 2.0 License.
+ * See the accompanying LICENSE file for terms.
+ */
+package com.yahoo.squidb.annotations.defaults;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation for explicitly specifying that a column has a default value of null. This also guarantees that null
+ * will appear in the model default values.
+ */
+@Target(ElementType.FIELD)
+public @interface DefaultNull {
```

Comment:
I know, hence the javadocs in this annotation -- this annotation is used to explicitly specify that a default value of null appears in the in-memory model default values. This is analogous to the current behavior of `@ColumnSpec(defaultValue = ColumnSpec.DEFAULT_NULL)`. SQLite uses a null default either way, but right now we need the annotation for the in-memory values. I have it on my roadmap to change this in a future PR as well, but it's out of scope for this PR.